src/pkg/bundle/deploy_test.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/stretchr/testify/require"
 	"github.com/zarf-dev/zarf/src/api/v1alpha1"
 	"github.com/zarf-dev/zarf/src/pkg/state"
-	zarfTypes "github.com/zarf-dev/zarf/src/types"
+	zarfState "github.com/zarf-dev/zarf/src/pkg/state"
```

Comment:
This is now imported twice. We should remove the unnamed import and refactor any references to it.